### ISSUE TITLE
With custom titlebar, menus work quite differently from before

### ISSUE THREAD

**Issue by yume-chan** [Microsoft/vscode#52285]
Issue Type: Bug

Testing the new custom titlebar implementation (#51422)

![image](https://user-images.githubusercontent.com/1330321/41583181-40adccac-73d6-11e8-9595-9e0ca7232ff4.png)

1. I just opened this menu, the first item now has keyboard focus. It used to require a <kbd>Down Arrow</kbd> to focus the first item. I think the current behavior is also reasonable and acceptable, just want to ask if it's intended.

However there is another minor display issue: the keyboard focus ring is too small. It should warp around the whole menu item, but now only the label part.

![image](https://user-images.githubusercontent.com/1330321/41583144-2f8776e4-73d6-11e8-9f55-681a82c3178c.png)

2. Now I moved the keyboard focus to the second item, and hover my cursor on the first item. So now I have two focuses? I don't think a program should have a mouse focus and a keyboard focus simultaneously.

3. Click "File" to open the menu, then click on "File" again, the menu should dismiss. It works most time, only except when I "click" via tapping on my Surface Pro's touch pad. It will quickly dismiss and reopen the menu.

![image](https://user-images.githubusercontent.com/1330321/41586048-59edbb84-73de-11e8-8e40-55d03ddac8f1.png)

4. The check marks after the label look strange, why not display them in front of labels like most programs' menus?

**Comment by sbatten** [comment]
@yume-chan Thank you for your feedback. Though this issue was created very early, this item now exists broken up in several other issues. I am going to close this one for simplicity.

**Comment by sbatten** [comment]
dupe of #52929
